src/test/java/posters/tests/smoke/RegisterTest.java

```diff
@@ -18,12 +15,11 @@
 import posters.dataobjects.User;
 import posters.flows.DeleteUserFlow;
 import posters.flows.OpenHomePageFlow;
-import posters.pageobjects.pages.browsing.HomePage;
-import posters.pageobjects.pages.user.LoginPage;
-import posters.pageobjects.pages.user.RegisterPage;
 import posters.tests.AbstractTest;
 
 /**
+ * Since john@doe.com is an already registered user in the DB, a RegisterTest with this would lead to an error.
```

Comment:
NB:The initial data base of Posters provides a default user to avoid a registration process. The used e-mail address is "john@doe.com". Using this user within the RegisterTest would lead to an expected error.